widgets/matplotlib/measurement/energy_spectrum.py

```diff
@@ -57,7 +57,11 @@ class MatplotlibEnergySpectrumWidget(MatplotlibWidget):
     """Energy spectrum widget
     """
     # By default, draw spectra lines with a solid line
-    default_linestyle = "-"
+    DEFAULT_LINESTYLE = "-"
+    MEASURED_SUM_SPECTRUM_LINE_STYLE = "dashed"
+    SIMULATED_SUM_SPECTRUM_LINE_STYLE = "dotted"
+    SUM_SPECTRUM_LINE_WIDTH = 2
+    Z_ORDER = 100
```

Comment:
An object with the highest [`zorder`](https://matplotlib.org/3.2.2/gallery/misc/zorder_demo.html) value is placed first. We want to place the sum spectra at the bottom, so the `zorder` should be negative.

Measured and simulated sum spectra could also have their own values, so that they are always in the same order.

```suggestion
    SIMULATED_SUM_SPECTRUM_Z_ORDER = -1
    MEASURED_SUM_SPECTRUM_Z_ORDER = -2
```

Reply:
OK